apps/package.json

```diff
@@ -26,12 +26,15 @@
     "build:stats": "node ./script/build-stats.js",
     "prestorybook": "curl -o build/package/css/application.css http://localhost:3000/assets/application.css",
     "storybook": "start-storybook -p 9001 -s build/package/",
-    "storybook:deploy": "./script/deploy-storybook.sh"
+    "storybook:deploy": "./script/deploy-storybook.sh",
+    "codemod:es6-modules": "./script/codemod-es6-modules.sh",
+    "codemod:move-file": "./script/codemod-move-file.sh"
```

Comment:
really we should upgrade to a more recent version of node :)

This is the place where people go to discover scripts, so I think if I put it anywhere else, it will never get used.